### DISCUSSION TITLE
Zappa + FastApi?

### DISCUSSION THREAD

**Discussion by dsmurrell** [fastapi/fastapi#7805] [Questions]
Is FastApi compatible with Zappa? I currently use Zappa and Flask together to deploy a rest API to AWS.

If not, what is the easiest way to deploy FastApi to lambda like cloud based serverless functions?

**Comment by nsidnev** [comment]
Well, a quick search showed that `zappa` is not yet able to work with ASGI, so you are unlikely to be able to use it to deploy `FastAPI` applications on AWS Lambda. 

But there is a [magnum](https://github.com/erm/mangum) project that you can try for these purposes. You can also see an example project [here](https://github.com/iwpnd/fastapi-aws-lambda-example) and there is a link to the guide in readme.

**Comment by dsmurrell** [comment]
Thanks, this is helpful, I will look into this!

**Comment by tiangolo** [comment]
Thanks for the help @nsidnev ! :cake: 

Thanks @dsmurrell for reporting back and closing the issue :+1:
